.sass-lint.yml

```diff
@@ -0,0 +1,103 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Set the formatter to 'html'
+  formatter: stylish
+# File Options
+files:
+  include: 'src/**/*.scss'
+# Rule Configuration
+rules:
+  # Extends
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+
+  # Mixins
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - 'respond-at-and-above'
+        - 'respond-between'
+        - 'respond-below'
+        - 'after'
+        - 'before'
+
+  # Line Spacing
+  one-declaration-per-line: 2
+  empty-line-between-blocks: 2
+  single-line-per-selector: 2
+
+  # Disallows
+  no-color-keywords: 2
+  no-color-literals: 0
```

Comment:
I think we can envision no one-offs, but there always are. I don't mind rolling them into a var, in the end. I see the benefit of naming to avoid the perception of "magic". 

@misscs?